initrd/bin/change-time

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#change time using hwclock and date -s
+
+clear
+
+echo -e -n "Please enter the date and time you wish to set\n"
```

Comment:
GMT/UTC and reference to pure HMTL5 url for user to look for GMT timezone clock with seconds like https://time.is/GMT or better